### DISCUSSION TITLE
SCTransform v2 and FindAllMarkers on a single sample (non-integrated dataset)

### DISCUSSION THREAD

**Discussion by SingleCellNewb** [satijalab/seurat#5605] [Q&A] [answered]
Hello everyone, 

I am quite new to the scRNASeq world, and recently I have started to analyze scRNASeq data using Seurat. I was wondering which assay, (SCT or RNA), should be used when invoking FindAllMarkers function on SCTv2 transformed data for a single sample. I have found some discussions regarding the use of the appropriate assay on SCTv1 transformed data and integration, but I am not sure about the SCTv2 transformed data and a single sample (no integration). I am sorry if this is an obvious question, but I could not find a definitive answer, and would not want to make a silly mistake.

Thanks!

**Comment by saketkc** [comment, marked as answer]
With v2, we recommend using the SCT assay for finding markers. You can see an example in this [vignette](https://satijalab.org/seurat/articles/sctransform_v2_vignette.html)

**Reply by aideenoneill** [reply]
Dear saketkc, 

Could I please check that with SCT V0.3.5, I can use the "SCT" assay for FindAllMarkers on an integrated object containing two objects that have been normalised using SCT.  

I can see that  SCT v0.3.5 is later than v2 but given that the version numbering is not chronological, and given that this is a critical decision, I would be grateful to check this.

Thanks very much.
